packages/smarthr-ui/src/components/AppHeader/components/desktop/Navigation.tsx

```diff
@@ -96,15 +97,16 @@ const buildDropdownMenu = (
   navigations: Array<ChildNavigation | ChildNavigationGroup>,
 ): ComponentProps<typeof AppNaviDropdownMenuButton>['children'] => (
   <>
-    {navigations.map((navigation) => {
+    {navigations.map((navigation, index) => {
       if ('elementAs' in navigation) {
-        const Component = navigation.elementAs
+        const { elementAs: Component, current, ...rest } = navigation
         // TODO: DropdownMenuItemを作成し、elementAsを渡せるようにする
         return (
           <Component
-            {...navigation}
-            key={navigation.children}
-            aria-current={navigation.current}
+            {...rest}
+            // key={navigation.children}
+            key={`${index}-${navigation.children.toString()}`}
+            aria-current={current}
             className={commonButton({
               current: navigation.current,
               className: navigation.className,
```

Comment:
```suggestion
              current,
              className,
```
nits: 分割代入で current を取り出しているので navigation から引き当てなくてよさそう！　className も 102行目で取り出してもいいかもですね。これはエラーの修正とは関係がないので、やらなくても問題ないです！

Reply:
見落としてました！ありがとうございます〜！